packages/server/router/emotionRouter.ts

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+import EmotionController from '../controllers/EmotionController';
+
+const emotionRouter = express.Router();
+
+emotionRouter.route('/topics/:id/emotion').get(EmotionController.getEmotionByTopicId);
```

Comment:
```suggestion
emotionRouter.get('/topics/:id/emotion', EmotionController.getEmotionByTopicId);
```
как будто проще выглядит, без лишнего